rector-migrate.php

```diff
@@ -2,6 +2,7 @@
 
 declare(strict_types=1);
 
+use Faker\Generator;
```

Comment:
We want to replace references to magic properties on `Faker\Generator`, not in `Generator`.

Missed this in #542.